packages/material-ui/src/Slider/Slider.d.ts

```diff
@@ -24,13 +24,13 @@ export type SliderTypeMap<
      * @default {}
      */
     classes?: SliderUnstyledTypeMap['props']['classes'] & {
-      /** Class name applied to the root element if `color="primary"`. */
+      /** Styles applied to the root element if `color="primary"`. */
       colorPrimary?: string;
```

Comment:
I think that "Class name" makes more sense in v5. The component does no longer apply any style with these classes cc @mnajdova.

Reply:
Agree, we should use "Class name" - another bullet point for me to check if it is updated in all components.

Reply:
I had missed that "Class name" was used for unstyled components. I'll revert that commit.

Reply:
Should it also be used for the styled components? (ahead of time of the whole migration to emotion)